ddp-workspace/projects/ddp-osteo/src/app/components/welcome/welcome.component.html

```diff
@@ -188,16 +188,12 @@ <h2 translate>
                         <img lazy-resource class="image" src="assets/images/partner-os-collaborative.svg"
                             [alt]="'HomePage.PartnersSection.ImagesAlts.OsCollaborative' | translate">
                     </a>
-                    <a href="http://www.targetcancerfoundation.org/" target="_blank">
-                        <img lazy-resource class="image" src="assets/images/partner-tcf.png"
-                            [alt]="'HomePage.PartnersSection.ImagesAlts.TCF' | translate">
+                    <a href="https://mcamf.org/" target="_blank">
+                        <img lazy-resource class="image" src="assets/images/partner-mcamf.png"
+                             [alt]="'HomePage.PartnersSection.ImagesAlts.MCAMF' | translate">
                     </a>
                 </div>
                 <div class="partners__group">
-                  <a href="https://mcamf.org/" target="_blank">
-                    <img lazy-resource class="image" src="assets/images/partner-mcamf.png"
-                         [alt]="'HomePage.PartnersSection.ImagesAlts.MCAMF' | translate">
-                  </a>
```

Comment:
Moved `partner-mcamf.png` up to the before `div` block in order to keep number of images unchanged at 3 per row.